wrappers/ortools.rb

```diff
@@ -315,9 +315,12 @@ def solve(vrp, job, thread_proc = nil, &block)
           ),
           rests: vehicle.rests.collect{ |rest|
             OrtoolsVrp::Rest.new(
-              time_windows: rest.timewindows.collect{ |tw|
-                OrtoolsVrp::TimeWindow.new(start: tw.start, end: tw.end || 2**56)
-              },
+              # Ortools can only deal with one and exactly one timewindow per rest :
+              time_windows:
+                rest.timewindows.any? ?
+                  rest.timewindows.collect{ |tw|
+                    OrtoolsVrp::TimeWindow.new(start: tw.start, end: tw.end || 2**56)
+                  } : [OrtoolsVrp::TimeWindow.new(start: 0, end: 2**56)], # Rests should always have a timewindow
```

Comment:
I still think that since the segfault is happening because of an error in optim-ortools ("de-referencing an uninitialized variable") and that there is an easy fix, we should fix it in optim-ortools. But I note this for myself, we can have this in the mean time.